src/containers/paper-canvas.css

```
@@ -1,7 +1,9 @@
 .paper-canvas {
-    width: 480px;
-    height: 360px;
+	top: 1px; /* leave room for the border*/
+	left: 1px;
```

Nit: these appear to be tabs

```suggestion
    top: 1px; /* leave room for the border*/
    left: 1px;
```